crates/core_arch/src/x86/bt.rs

```diff
@@ -0,0 +1,105 @@
+#[cfg(test)]
+use stdsimd_test::assert_instr;
+
+/// Return the bit at index `b` of 32-bit integer `a`.
+#[inline]
+#[cfg_attr(test, assert_instr(btl))]
+#[unstable(feature = "bittest", issue = "0")]
```

Comment:
probably want to name the feature `simd_x86_bittest` - this will need support upstream.